node/manifest.test.ts

```diff
@@ -30,6 +30,17 @@ test('Generates a manifest with different bundles', () => {
   expect(manifest.bundler_version).toBe(env.npm_package_version as string)
 })
 
+test('Generates a manifest with display names', () => {
+  const functions = [{ name: 'func-1', path: '/path/to/func-1.ts' }]
+  const declarations = [{ function: 'func-1', name: 'Display Name', path: '/f1/*' }]
```

Comment:
Could you maybe add a second function and declaration, without a `name`, to capture the fact that we'll gracefully handle a function without one, and that it's going to simply be missing from the manifest?

Reply:
I didn't do this because the other tests already do this: https://github.com/netlify/edge-bundler/blob/main/node/manifest.test.ts#L7-L31
Is that enough or should I still add one more case?

Reply:
Good question. Personally, I tend to encapsulate all cases in the test related to the feature, as opposed to relying on the assertions from other unrelated tests. The only reason being that if you remove or change one of those tests in the future, it might not be immediately obvious to you that you're removing coverage for this other feature somewhere else in the codebase.

Totally up to you, though.

Reply:
okay added it to the test